pkg/dbc/analysis/passes/uniquemessageids/analyzer.go

```diff
@@ -0,0 +1,47 @@
+package uniquemessageids
+
+import (
+	"go.einride.tech/can/pkg/dbc"
+	"go.einride.tech/can/pkg/dbc/analysis"
+)
+
+//nolint:gochecknoglobals
+var globalMessageIDs map[dbc.MessageID]struct{}
```

Comment:
@Jassob Not sure about this, it will produce a lot of errors.. Would be nice if we could emit warnings only for this.

Reply:
Or if we add a "severity" to the `Pass` struct perhaps.

Reply:
I skipped the global check.